src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_help.py

```diff
@@ -181,37 +187,40 @@
 short-summary: Describe a helm chart in an Azure Container Registry.
 examples:
   - name: Show all versions of a helm chart in an Azure Container Registry
-    text: >
+    text: >-
         az acr helm show -n MyRegistry mychart
   - name: Show a helm chart version in an Azure Container Registry
-    text: >
+    text: >-
         az acr helm show -n MyRegistry mychart --version 0.3.2
+  - name: Describe a helm chart in an Azure Container Registry. (autogenerated)
+    text: az acr helm show --name MyRegistry mychart
```

Comment:
Same as the above example... but less specific because it doesn't demonstrate the use of the version field.